core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.ts

```diff
@@ -36,9 +37,10 @@ export class AIContentPromptComponent {
             this.aiContentService
                 .getIAContent(textPrompt)
                 .pipe(catchError(() => of(null)))
-                .subscribe(() => {
+                .subscribe((response) => {
                     this.isFormSubmitting = false;
                     this.formSubmission.emit(true);
+                    this.aiResponse.emit(response);
```

Comment:
You can unify these two emitter. They are being called in the same place and only once.

```suggestion
                    this.aiResponse.emit(response);
```